islandora_internet_archive_bookreader.drush.inc

```diff
@@ -8,12 +8,12 @@
 /**
  * The Internet Archive Bookreader plugin URI.
  */
-define('IABOOKREADER_DOWNLOAD_URI', 'https://github.com/Islandora/internet_archive_bookreader/archive/master.zip');
+define('IABOOKREADER_DOWNLOAD_URI', 'https://github.com/internetarchive/bookreader/archive/2.0.2.zip');
 
 /**
- * The original extracted directory frome the zip file.
+ * The original extracted directory from the zip file.
  */
-define('IABOOKREADER_ORIGINAL_DIR', 'internet_archive_bookreader-master');
+define('IABOOKREADER_ORIGINAL_DIR', 'internet_archive_bookreader-2.0.2');
```

Comment:
This is wrong. original dir should be 'bookreader-2.0.2' @patdunlavey do you have time to make a pull here?

Reply:
Fixed via https://github.com/Islandora/islandora_internet_archive_bookreader/pull/133/commits/49eb72d1feda9f64731acb3218a5c59146461a21